src/_scss/pages/bulkDownload/archive/_archiveForm.scss

```diff
@@ -1,6 +1,12 @@
 .award-data-archive-form {
     background-color: #F3FAFC;
     padding: rem(15) rem(5);
+
+    .form-title__wrapper {
```

Comment:
Is this wrapper/ the flex properties necessary? We could add `padding-bottom: rem(10);` to the `.form-title` element for spacing.